src/Shared/Components/SelectPicker/type.ts

```diff
@@ -190,6 +192,8 @@ export type SelectPickerProps<OptionValue = number | string, IsMulti extends boo
          * @default false
          */
         fullWidth?: boolean
+        isCreatableSingleSelect?: boolean
```

Comment:
suggestion: add ts doc

Reply:
Instead of exposing this new prop, we should move the isCreatable out of the multiSelectProps at the base level and use CreatableSelect instead of the React Select in the component part.